### ISSUE TITLE
Check that we are pulling all available NGWOS data

### ISSUE THREAD

**Issue by lekoenig** [USGS-R/drb-inland-salinity-ml#9]
Check that our SpC data download from NWIS daily sites (service = "dv") includes all available NGWOS sensor sites. 

@jds485 & @aappling-usgs, are you aware of a published list of NGWOS stations in the DRB? I found a list of 124 sites within the IIDD --> Project - Modeling Streams docs, but I'm not sure if it's still up to date.

**Comment by jds485** [comment]
No, I don't know of a list with those stations. I'd ask Joel Blomquist.

**Comment by jds485** [comment]
@lekoenig will this issue be resolved with the PR #19, or are you waiting on further info about NGWOS sites? If 19 will close the issue, you can link the pull request to this (panel on right).

**Comment by lekoenig** [comment]
This issue will be closed with PR#19. I don't incorporate this check as a target within the pipeline, but we are capturing all 68 NGWOS SC sites within the DRB. I just linked this issue to the pull request, thanks for that suggestion!
